fe/fe-core/src/main/java/com/starrocks/server/WarehouseManager.java

```diff
@@ -146,10 +151,7 @@ public List<Long> getAllComputeNodeIds(long warehouseId) {
     }
 
     private List<Long> getAllComputeNodeIds(long warehouseId, long workerGroupId) {
-        Warehouse warehouse = idToWh.get(warehouseId);
```

Comment:
Why do we need to delete the existence check of warehouse?

Reply:
There is check in the getWarehouse function